tools/testkit/src/lib.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+use fvm_integration_tests::dummy::DummyExterns;
+use fvm_integration_tests::tester::{Account as TAccount, Tester};
+use fvm_ipld_blockstore::MemoryBlockstore;
+use fvm_shared::state::StateTreeVersion;
+use fvm_shared::version::NetworkVersion;
+
+pub mod bundle;
+pub mod fevm;
+
+/// Execution options
+pub struct ExecutionOptions {
+    /// Enables debug logging
+    pub debug: bool,
+    /// Enables gas tracing
+    pub trace: bool,
+    /// Enabls events
+    pub events: bool,
+}
+
+pub type BasicTester = Tester<MemoryBlockstore, DummyExterns>;
+pub struct Account {
+    pub account: TAccount,
+    pub seqno: u64,
+}
+
+pub fn new_tester(bundle_path: String) -> BasicTester {
+    let blockstore = MemoryBlockstore::default();
+    let bundle_cid = match bundle::import_bundle(&blockstore, bundle_path.as_str()) {
+        Ok(cid) => cid,
+        Err(what) => {
+            panic!("error loading bundle: {}", what);
+        }
+    };
+    Tester::new(
+        NetworkVersion::V18,
+        StateTreeVersion::V5,
+        bundle_cid,
+        blockstore,
+    )
+    .unwrap_or_else(|what| {
+        panic!("error creating execution framework: {}", what);
+    })
+}
+
+pub fn create_account(tester: &mut BasicTester) -> Account {
+    let accounts: [TAccount; 1] = tester.create_accounts().unwrap();
+    Account {
+        account: accounts[0],
+        seqno: 0,
+    }
+}
+
+pub fn create_accounts<const N: usize>(tester: &mut BasicTester) -> [Account; N] {
+    let accounts: [TAccount; N] = tester.create_accounts().unwrap();
+    accounts.map(|a| Account {
+        account: a,
+        seqno: 0,
+    })
+}
+
+pub fn prepare_execution(tester: &mut BasicTester, options: &ExecutionOptions) {
```

Comment:
What does this do? This doesn't seem like something that should be public.

(I'm not asking you to _tell_ me what this does, I'm pointing out that we have a super-generic public method called `prepare_execution` with no documentation that likely belongs on the test framework itself.)

Reply:
FWIW I moved it [inside my test driver](https://github.com/filecoin-project/ref-fvm/pull/1507/files#diff-0b38f777658656ae3e7df7b053d6daa49846b1af6f559bfd60c726b0d58cd904R138). Turns out I didn't understand when exactly we should call this.